### PR TITLE
[MRG] revise command-line docs for `sourmash sig` subcommands

### PR DESCRIPTION
**Pull request by ctb**
Building on https://github.com/sourmash-bio/sourmash/pull/1714.

Updates the text for `sig intersect` and `split`.

### CODE REVIEW

**Comment by ctb** [comment]
@sourmash-bio/devs any thoughts on what we should put into usage strings? Now's the time to make suggestions :)

For example,
```
sourmash sig cat -h
```

now displays
```

== This is sourmash version 0.0.0. ==
== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==

usage:

    sourmash sig cat *.sig -o combined.sig

sourmash sig cat concatenates signature files.

For example,

    sourmash signature cat file1.sig file2.sig -o all.sig

will combine all signatures in `file1.sig` and `file2.sig` and put them
in the file `all.sig`.

concatenate signature files

positional arguments:
  signatures

optional arguments:
  -h, --help            show this help message and exit
  --from-file FROM_FILE 
                        a text file containing a list of files to load
                        signatures from
...
```

**Comment by taylorreiter** [comment]
That message is super helpful! It may be nice to add to the documentation if this works for multiple molecule types/multiple k types/multiple scaleds etc, and if smooshing this info will cause downstream issues or if it's all fine.

**Comment by ctb** [comment]
> That message is super helpful! It may be nice to add to the documentation if this works for multiple molecule types/multiple k types/multiple scaleds etc, and if smooshing this info will cause downstream issues or if it's all fine.

hot take thought - what do you think about designing some generic statements that we pepper throughout, like:

* this command requires that all sketches be compatible (ksize, moltype) and you can specify -k and --dna/etc on the command-line to choose them
* this command works with all input sketches; specific types of sketches can be selected with -k and --dna/etc.

? then we can make it consistent... I'll have to see if that's plausible with the commands we have, of course, but I suspect it is (since we mostly use common code)

**Comment by taylorreiter** [comment]
> hot take thought - what do you think about designing some generic statements that we pepper throughout, like:
> 
>     * this command requires that all sketches be compatible (ksize, moltype) and you can specify -k and --dna/etc on the command-line to choose them
> 
>     * this command works with all input sketches; specific types of sketches can be selected with -k and --dna/etc.
> 
> 
> ? then we can make it consistent... I'll have to see if that's plausible with the commands we have, of course, but I suspect it is (since we mostly use common code)

Ya I think that sounds great. The messages you have above sound good to me too, they're clear and give me a sense of how to do what I want to achieve e.g. my particular use case (whatever that may be)

**Comment by ctb** [comment]
Ready for review & merge @sourmash-bio/devs

File: src/sourmash/cli/sig/intersect.py [hunk 1]
```diff
@@ -18,6 +18,8 @@
 in any signatures will be ignored and the output signature will have
 `track_abundance` turned off.
 
+Note: `intersect` only creates one output file, with one signature in it.
```
Comment (ccbaumler) [inline src/sourmash/cli/sig/intersect.py:21]:
```suggestion
Note: `intersect` only creates one output file, with one consolidated signature from the input files.
```

File: src/sourmash/cli/sig/intersect.py [hunk 2]
```diff
@@ -18,6 +18,8 @@
 in any signatures will be ignored and the output signature will have
 `track_abundance` turned off.
```
Comment (ccbaumler) [inline src/sourmash/cli/sig/intersect.py]:
```suggestion
`track_abundance` turned off. See `sourmash signature flatten` for more details.
```

**Review by ccbaumler — APPROVED** [review]
See suggestions above for more specific language. Otherwise, looks good to me.

File: src/sourmash/cli/sig/intersect.py (hunk 1 above)
Comment (ctb) [reply src/sourmash/cli/sig/intersect.py:21]: I'm not sure what consolidated here would mean so I'm going to ignore this suggestion ;)

**Review by ghost — COMMENTED** [review]
pisahkan

**Pull request merged**